src/applications/disability-benefits/all-claims/config/form0781/index.js

```diff
@@ -34,6 +36,13 @@ export const form0781PagesConfig = {
     uiSchema: workflowChoicePage.uiSchema,
     schema: workflowChoicePage.schema,
   },
+  manualUploadPage: {
+    path:
```

Comment:
Probably only need the subpath, `additional-forms/mental-health-statement/upload`, and not the full path here, when this page is fully implemented.